catalogs/qnl.yaml

```diff
@@ -1,10 +1,11 @@
 metadata:
   version: 1
 sources:
-  qnl:
+  british_library:
     driver: oai_xml
     args:
```

Comment:
This collection is failing; the DAG isn't building correctly. I get the 'qnl_None_harvest', 'qnl_None_metadata' labels on the tasks in Airflow. I'm not sure why but wonder if its because there is not OAI set. This collection doesn't support sets, so has to be harvested from ListRecords. https://api.qdl.qa/api/oaipmh?verb=ListSets (need to be whitelisted to see)